readme.md

```diff
@@ -171,19 +176,20 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
 
 * **Your Answer:**
-
+add a try catch block and use the error.message to give a more descriptive error
 ---
 
 - [ ] With Mongo, it is simple to create complex data structures. Add a `characters` field to your Series model that is an array of objects. Each object should have a `name` and `image_url`. Only the `name` field on the character should be required. _Note: Don't forget to change your select statements to include the `characters` field!_
 
 * **Question:** Take a look at the response from making a new series with a character. What field was created that you did not have to define?
 
 * **Your Answer:**
+an _id field was created for each character
 
 * **Question:** With the current routes that we have, how would you upate the name of a character in a series?
 
 * **Your Answer:**
-
+the PUT route could update the character
```

Comment:
The idea here is that you will have to go _through_ the Series model in order to update characters.